server/src/routes/auth.js

```diff
@@ -0,0 +1,55 @@
+import cors  from 'cors';
+import bodyParser from 'body-parser';
+import database from '../database/db';
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import cookieParser from 'cookie-parser';
+import config from '../../config';
+import { SESSION_COOKIE } from '../app';
+
+
+
+
+export default function(router) {
+    router.use(cors())
+    router.use(bodyParser.json())
+    router.use(cookieParser())
+
+    router.post('/register', (req, res)=> {
+    const saltRounds = 10;
+    const salt = bcrypt.genSaltSync(saltRounds);
+    const {fname, lname, email, password} = req.body.users;
+    const hash = bcrypt.hashSync(password, salt);
+
+    database.query('INSERT INTO users(fname, lname, email, password) VALUES($1, $2, $3, $4) returning userid',
```

Comment:
Again, indentation.

Don't shy away from creating variables for your arrays and objects. If you store them as such, your function invocations are easier to format.

```js
const jwtParams =   {
  secure: config.get('HTTPS'),
  maxAge: 7200000,
  httpOnly: true
}

jwt.sign(sessionUser, jwtParams)
```